app/views/lists/_index.html.erb

```diff
@@ -1,25 +1,35 @@
-<% lists.each do |list| %>
-  <div class="card">
-    <% if user_signed_in? %>
-      <%= render partial: 'votes/toggle_like', locals: { votable: list } %>
-    <% end %>
-    <span class="info">
-      <h2><%= list.name %></h2>
-      <p><%= list.description %></p>
-      <p>Tags:<i><%= list.tag_list %></i></p>
-    </span>
-    <span class="content">
-      <%= render partial: 'lists/references', locals: { list: list } %>
-    </span>
-    <span class="options">
-      <ul>
-        <li><%= link_to 'Show', list %></li>
+<% lists.each_slice(3) do |three_lists| %>
```

Comment:
Sounds good.

On Fri, Aug 12, 2016 at 9:00 PM, wsf246 <notifications@github.com> wrote:

> In app/views/lists/_index.html.erb
> <https://github.com/briankung/projectcredo/pull/52#discussion_r74679584>:
>
> > -  <div class="card">
> > -    <% if user_signed_in? %>
> > -      <%= render partial: 'votes/toggle_like', locals: { votable: list } %>
> > -    <% end %>
> > -    <span class="info">
> > -      <h2><%= list.name %></h2>
> > -      <p><%= list.description %></p>
> > -      <p>Tags:<i><%= list.tag_list %></i></p>
> > -    </span>
> > -    <span class="content">
> > -      <%= render partial: 'lists/references', locals: { list: list } %>
> > -    </span>
> > -    <span class="options">
> > -      <ul>
> > -        <li><%= link_to 'Show', list %></li>
> > +<% lists.each_slice(3) do |three_lists| %>
>
> That makes sense. I haven't found any sensible solutions. maybe let's pair
> code sometime to figure it out.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/briankung/projectcredo/pull/52/files/b653b00e5e4f23efb6cb495404de572981f0e34c#r74679584>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACtGxwqw8p4JOwrd_S9MPhusa24MTVqXks5qfSUqgaJpZM4JjW-e>
> .
>



-- 
Brian Kung
http://about.me/briankung/
<http://www.facebook.com/brian.kung> <http://www.linkedin.com/in/briankung>
<http://www.twitter.com/briankung>

BE A UNICORN!